recipes/proxy.rb

```diff
@@ -25,12 +25,16 @@
 
 file "#{node['nginx']['dir']}/conf.d/jenkins.cert" do
   content ssl['client']['cert']
-  mode 0644
+  owner 'root'
```

Comment:
I think it may be better to configure the user as well. Can we use a more descriptive name for root_group? maybe something like 'jenkins_cert_user', 'jenkins_cert_group'. If I am right, root_group is not defined in default attributes yet?